services/discordService.js

```diff
@@ -64,8 +64,20 @@ const addRoleToUser = async (userid, roleid) => {
   return response;
 };
 
+const removeRoleFromUser = async (roleId, discordId) => {
+  const authToken = await generateAuthTokenForCloudflare();
+  const data = await fetch(`${DISCORD_BASE_URL}/roles`, {
+    method: "DELETE",
+    headers: { "Content-Type": "application/json", Authorization: `Bearer ${authToken}` },
+    body: JSON.stringify({ userid: discordId, roleid: roleId }),
+  });
```

Comment:
can we add try catch block for both of these operation,

Reply:
error handling will be done in consuming function